src/acl/Acl.cc

```diff
@@ -211,7 +237,16 @@ Acl::Node::ParseAclLine(ConfigParser &parser, Node ** head)
         return;
     }
 
-    xstrncpy(aclname, t, ACL_NAME_SZ);
+    SBuf aclname(t);
+    CallParser(ParsingContext::Pointer::Make(aclname), [&] {
+        ParseNamed(parser, head, aclname.c_str()); // TODO: Convert Node::name to SBuf
```

Comment:
IMHO, this c-string-to-SBuf conversion does not qualify for a "performance regression" XXX because the now-removed code was also copying the parsed token (probably for no good reason). All this noise (and remaining copying in Acl::Node constructor) will go away when we convert Acl::Node::name to SBuf, but that change requires a dedicated PR. Said that, if others insist on this PR adding an XXX comment here, I will do that without arguing.